templates/liquid/Measure.liquid

```diff
@@ -127,7 +127,7 @@
         {% if Measure.author.exists() %}
```

Comment:
If we're changing what we're displaying, we should also change what we're conditioning the display on, this should probably now be Measure.author.name.exists() yes?